models/chain-information.ts

```diff
@@ -0,0 +1,21 @@
+export interface ChainInformation {
+  chainName: string
+  description: string
+  platformChainId: string
+  subnetId: string
+  vmId: string
+  vmName: string // EVM, BITCOIN, ETHEREUM
+  rpcUrl: string // uri
+  wsUrl?: string // uri
+  multicallAddress?: string // ChecksumAddress
```

Comment:
Are there any other utility addresses that we may add in the future?  Wondering if we want to nest this in a `utilityAddress` object.  Not a big deal though.

Reply:
Yep this works for me.  There are currently no other utility addresses, but I certainly foresee more in the future.  I like the idea 👍